core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -452,17 +452,50 @@ oppia.controller('CreatorDashboard', [
                 oldContent: $scope.oldContent
               });
             };
+            $scope.saveSuggestion = function() {
+              $uibModalInstance.close({
+                action: ACTION_EDIT_SUGGESTION,
+                newSuggestionHtml: $scope.suggestionData.newSuggestionHtml,
+                summaryMessage: $scope.summaryMessage,
+                stateName: $scope.stateName,
+                suggestionType: $scope.suggestionType,
+                oldContent: $scope.oldContent
+              });
+            };
           }
         ]
       }).result.then(function(result) {
         var RESUBMIT_SUGGESTION_URL_TEMPLATE = (
           '/suggestionactionhandler/resubmit/<suggestion_id>');
+        var EDIT_SUGGESTION_URL_TEMPLATE = (
+          '/suggestionactionhandler/edit/<target_id>/<suggestion_id>');
         var HANDLE_SUGGESTION_URL_TEMPLATE = (
           '/suggestionactionhandler/<target_type>/<target_id>/<suggestion_id>');
 
         var url = null;
         var data = null;
-        if (result.action === 'resubmit' &&
+        if (result.action === 'edit' &&
+            result.suggestionType === 'edit_exploration_state_content') {
```

Comment:
Do we have any other suggestion type? If yes can we use an oppia.constant for this as I can see this in other place too.

Reply:
i don't think so.
Are there any @nithusha21